source/CardGames.py

```diff
@@ -155,4 +180,28 @@ def dealCards(self, numCards: int, players: "list[Player]"):
 
   def resetDeck(self):
     self.deck.reset()
-    self.deck.shuffle()
\ No newline at end of file
+    self.deck.shuffle()
+
+
+# has_pair function---------------------------------------------------------------
```

Comment:
This section doesn't really relate to the pull request.  It probably should have been its own PR in the next sprint.  (Nitpicking because I have to say something)

Reply:
Just realized that there is a pull request for this but it is also in this due to rebasing

Reply:
lol no worries! Thank you for the input I will keep things seperate next time